wlauto/workloads/gmail/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -153,9 +153,9 @@ public void attachFiles() throws Exception {
             Timer result = new Timer();
             result.start();
 
-            clickUiObject(attachIcon, timeout);
+            attachIcon.click();
```

Comment:
The documentation for clickAndWaitForNewWindow states:
> Waits for window transitions that would typically take longer than the usual default timeouts.

I don't think minor popup menu qualify. This should save some time too.